examples/tutorials/Part 0 - Installation Guide.md

```diff
@@ -0,0 +1,228 @@
+## 0.2 Mac OS
+-------------------
+
+#### 1. Install python
+
+In order to install python on MacOs, you first need to install **Homebrew** the famous package manager.
+
+Start by opening a terminal and type the following:
+
+<font color='red'>$</font> `xcode-select --install`
+
+<font color='red'>$</font> `/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"`
+
+This will install **Homebrew**. Then, install python (version 3.6 or above) by running:
+
+<font color='red'>$</font> `brew install python3`
+
+#### 2. Install pytorch 1.0
+
+You should first get the installation command here:
+
+https://pytorch.org/get-started/locally/. (use the pip option)
+
+Then run that command in the open terminal.
+
+#### 3. Clone the PySyft repo from Github
+
+Just type
+
+<font color='red'>$</font> `git clone https://github.com/OpenMined/PySyft.git`
+
+In the terminal
+
+#### 4. Install PySyft
+
+First, you need to enter the PySyft folder you cloned earlier by typing:
+
+<font color='red'>$</font> `cd PySyft`
+
+in the terminal. Then, run this to install the package:
+
+<font color='red'>$</font> `python setup.py install`
+
+You can  test your installation by running:
+
+<font color='red'>$</font> `python setup.py test`
+
+
+-----------------------------------
+## 0.3 Windows
+------------------------------
+
+#### 1. install python
+
+PySyft requires python version 3.6 or above.
+
+Here is the link to install python https://www.python.org/downloads
+
+#### 2. add ***python*** and ***pip*** to the `PATH` environment variable
+
+First, you need to find the folder path to which `python.exe` and `pip3` were installed.
+
+In my case it is:
+
+>C:\Users\alan\AppData\Local\Programs\Python\Python37
+
+for the former, and:
+
+>C:\Users\alan\AppData\Local\Programs\Python\Python37\Scripts
+
+for the latter. So I will add both paths to the PATH environment variable.
+
+In order to do that, go to **Run** and type `sysdm.cpl` and press **Enter**. The following window should open:
+
+
+
+<img src='./images/sysdmcpl.png'>
+
+
+
+Click on the `Environment Variables...` button in the bottom-right corner. The following window should appear:
+
+
+<img src='./images/sysdmcpl2.png'>
+
+
+
+Select the **Path** row in the `System varibles` section as in the above screenshot and click on `Edit...`
+
+In the window that opens, click on `New` and add the installation paths for `python.exe` and `pip`. Here is an example:
+
+
+
+<img src='./images/sysdmcpl3.png'>
+
+
+Click on `OK`.
+
+#### 3. Install pytorch 1.0
+
+You should first get the installation command here:
+
+https://pytorch.org/get-started/locally/. (use the pip option)
+
+Then open the command prompt by going to **Run** and typing *cmd*, type or paste the pytorch installation command you got from the above link and press **Enter**.
+
+#### 4. Install Git
+
+Here is the download link for Git on windows: https://git-scm.com/download/win
+
+#### 5. Install Microsoft Build tools
```

Comment:
Do you actually need the build tools? Full disclaimer, I do not currently have a windows system to test this on

Reply:
It is actually necessary to get the C++ compiler to my understanding. I couldn't install PySyft without this step